src/libtoast/src/toast_math_rng.cpp

```diff
@@ -29,13 +29,13 @@ void toast::rng_dist_uint64(size_t n, uint64_t key1, uint64_t key2,
         #pragma omp simd
         for (size_t i = 0; i < n; ++i) {
             data[i] = rng(
-                RNG::ctr_type({{counter1, counter2 + i} }),
+                RNG::ctr_type({{counter1, counter2 + i}}),
```

Comment:
Lots of uncrustify noise, but you mentioned that `format_source.sh` gave an error on MacOS.  I'll look into that.  For now we can live with some noise.